server/GameOn/GameOn.Tournaments/Calculators/EloScoreCalculator.cs

```diff
@@ -10,7 +10,29 @@ public class EloScoreCalculator : IScoreCalculator
     {
         public ScoreResult[] Calculate(ScoreResult[] scores, string winnerId)
         {
-            throw new NotImplementedException();
+            ScoreResult playerOne = scores[0];
+            ScoreResult playerTwo = scores[1];
+
+            int outcome = 1;
+
+            if (winnerId == playerOne.PlayerId)
+                outcome = 0;
+
```

Comment:
Not essential but C# has a conditional expression syntax if you are interested? e.g.

```csharp
int outcome = winnerId == playerOne.PlayerId ? 0 : 1;
```